example/single/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 add_executable(single-cell EXCLUDE_FROM_ALL single.cpp)
 add_dependencies(examples single-cell)
-target_link_libraries(single-cell PRIVATE arbor arborenv arbor-sup)
+target_link_libraries(single-cell PRIVATE arbor arborenv arbor-sup arbor-private-deps arbor-private-headers)
```

Comment:
We should avoid having the example code rely upon the private headers. Is this an artefact of development and not needed anymore?

Reply:
Fixed. Indeed, a development artifact.